tests/php/src/Admin/AfterActivationSiteScanTest.php

```diff
@@ -276,7 +276,18 @@ function ( $caps, $cap ) {
 	 * @covers ::get_amp_compatible_plugins_url
 	 */
 	public function test_get_amp_compatible_plugins_url() {
-		wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) );
+		if ( is_multisite() ) {
+			$user_id = self::factory()->user->create(
+				[
+					'role' => 'administrator',
+				]
+			);
+
+			grant_super_admin( $user_id );
+			wp_set_current_user( $user_id );
+		} else {
+			wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) );
+		}
```

Comment:
Since this same logic is being repeated in multiple files, I suggest factoring it out into a [shared trait](https://github.com/ampproject/amp-wp/tree/develop/tests/php/src/Helpers) or else to put in the base [`TestCase`](https://github.com/ampproject/amp-wp/blob/develop/tests/php/src/TestCase.php).